skrub/datasets/_utils.py

```diff
@@ -1,5 +1,18 @@
+from os import environ
 from pathlib import Path
 
+from loguru import logger
+
+DATA_HOME_ENVAR_NAME = "SKRUB_DATA_DIRECTORY"
+DATA_HOME_ENVAR = environ.get(DATA_HOME_ENVAR_NAME)
+
+if DATA_HOME_ENVAR and (path := Path(DATA_HOME_ENVAR)).is_absolute():
+    DATA_HOME_DEFAULT = path
+else:
+    DATA_HOME_DEFAULT = Path("~").expanduser() / "skrub_data"
+
+logger.debug(f"Setting `DATA_HOME_DEFAULT` to '{DATA_HOME_DEFAULT}'.")
```

Comment:
we don't really use logging in the rest of the library. at some point we should probably do it in a systematic way, but for now you can just use `print` or maybe write to `sys.stderr`

Reply:
I agree with you but that's kind of a separate issue